services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -579,8 +579,9 @@ private string GetItemSourceCode(IngestModel ingest, string paperName, Dictionar
             }
             else
             {
+                var defaultSource = ingest.GetConfigurationValue("defaultSource");
                 _logger.LogWarning("Paper '{paperName}' not in configuration string for ingest '{ingest.Name}'.", paperName, ingest.Name);
```

Comment:
If they've configured a default source then it will use it.  There is no reason to log a warning in this case.  Only log the warning if they have not configured a default source and one wasn't found.

Reply:
Fixed.